src/sdk/staking.ts

```diff
@@ -234,15 +233,15 @@ async function metaMyStake(web3: Web3, address: string, account: string, isV2: b
             reward: rewardG$,
             reward$: {
                 claimed: CurrencyAmount.fromFractionalAmount(
-                    DAI,
+                  G$MainNet,
                     rewardUSDC.claimed.numerator,
                     rewardUSDC.claimed.denominator
-                ),
+                ) as CurrencyAmount<Currency>,
```

Comment:
is that casting you added everywhere necessary?

Reply:
no, apparently not